agent-inject/agent/annotations.go

```diff
@@ -231,26 +231,32 @@ const (
 	// will exit on template render failures once it has exhausted all its retry
 	// attempts. Defaults to true.
 	AnnotationTemplateConfigExitOnRetryFailure = "vault.hashicorp.com/template-config-exit-on-retry-failure"
+
+	// AnnotationTemplateConfigStaticSecretRenderInterval
+	// If specified, configures how often Vault Agent Template should render non-leased secrets such as KV v2.
+	// Defaults to 5 minutes.
+	AnnotationTemplateConfigStaticSecretRenderInterval = "vault.hashicorp.com/template-config-static-secret-render-interval"
```

Comment:
I think this is over the 63-character limit for annotation names: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

So we'll need to pick something shorter. Perhaps `template-static-secret-render-interval`?

Reply:
Thanks for the hint, @tvoran, working on the fix